src/type/matrix/DenseMatrix.js

```diff
@@ -231,7 +231,7 @@ export const createDenseMatrixClass = /* #__PURE__ */ factory(name, dependencies
     }
 
     const isScalar = index.isScalar()
-    if (isScalar) {
+    if (isScalar && !config.predictable) {
```

Comment:
That makes sense. Is it easy to remove the `config.predictable` support from this PR? Then it would be nice indeed to take this one step at a time and be able to think through `config.predictable` a bit more in detail.

Reply:
Closing this conversation, `config.predictable` is removed for now.